code/modules/mob/mob.dm

```diff
@@ -1237,3 +1237,6 @@ var/list/slot_equipment_priority = list( \
 
 	attack_log += new_log
 	last_log = world.timeofday
+
+/mob/proc/is_literate()
+	return 0
```

Comment:
Boolean

Reply:
Yeh, there are a lot more, I kind of gave up after a while.